src/game/frontend/submenus/World/weather.hpp

```diff
@@ -0,0 +1,34 @@
+#include <game/rdr/Natives.hpp>
+inline const char* WeatherTypes[]{
```

Comment:
maybe turn this into an unordered_map and i don't believe the inlining is required I could be wrong though

Reply:
1. Keep the array
2. Inline is required (not here in this specific case but it's a good thing to remember due to how header files work)

Reply:
Okay, I will remeber the inlining. I am curious, why would an array be better here? I guess an array and unordered_map are pretty close to the same thing.